tpe/lib/src/Utils.cc

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include "Utils.hh"
+
+namespace ignition {
+namespace physics {
+namespace tpelib {
+
+//////////////////////////////////////////////////
+math::AxisAlignedBox transformAxisAlignedBox(
```

Comment:
Could we add more documentation on the math behind this function? I find it a bit hard to understand by just reading the code.

Reply:
added more doc and testing linker fix in 558cbb7